src/modules/ekf2/EKF/mag_3d_control.cpp

```diff
@@ -45,13 +45,12 @@ void Ekf::controlMag3DFusion(const magSample &mag_sample, const bool common_star
 
 	resetEstimatorAidStatus(aid_src);
 
-	const bool wmm_updated = (_wmm_gps_time_last_set > aid_src.time_last_fuse);
+	const bool wmm_updated = (_wmm_gps_time_last_set >= aid_src.time_last_fuse); // WMM update can occur on the last epoch, just after mag fusion
```

Comment:
I'm wondering if we even need this given the updated WMM is immediately used in by limitDeclination in fuseMag(). Something to review later.